build/build.ps1

```diff
@@ -16,6 +16,16 @@ if ($Env:ENABLE_NATIVE -ne "false") {
     Write-Host "Skipping build of native simulator because ENABLE_NATIVE variable is set to: $Env:ENABLE_NATIVE."
 }
 
+if ($Env:ENABLE_QIRRUNTIME -ne "false") {
+    $qirRuntime = (Join-Path $PSScriptRoot "../src/Qir/Runtime")
+    & "$qirRuntime/build-qir-runtime.ps1"
+    if ($LastExitCode -ne 0) {
+        $script:all_ok = $False
+    }
+} else {
+    Write-Host "Skipping build of qir runtime because ENABLE_QIRRUNTIME variable is set to: $Env:ENABLE_QIRRUNTIME"
+}
```

Comment:
(Minor)
I assume that if the env var `ENABLE_QIRRUNTIME`  
is not set
or is set to any value other than `false` (lowercase)
then the QIR Runtime will be built. Is that a correct assumption?

Reply:
Yes, that is correct. This an inversion of the previous behavior, where it only built if the variable was set to "true" and so would default to not building QIR Runtime. This was updated in #566 .